src/core/router/identity.cc

```diff
@@ -730,27 +730,23 @@ Keys CreateRandomKeys() {
 IdentHash CreateRoutingKey(
     const IdentHash& ident) {
   std::uint8_t buf[41];  // ident + yyyymmdd
-  memcpy(buf, (const std::uint8_t *)ident, 32);
+  std::memcpy(buf, (const std::uint8_t *)ident, 32);
```

Comment:
This cast is unnecessary because of `Tags` functor.

Reply:
Ok, I'll remove it in the next refactor, if I keep the `memcpy`s.